Sample apps update (#888)
Macrometacorp/docs: docs/apps/sole-factory.md

---
sidebar_position: 1

> title: Macrometa Sole Factory

Reviewer: I'm confused by the naming. The first two seemed to be named based on features you are demonstrating, this one is based on what it is. Be consistent.

---

## Macrometa GDN

| **Platform**                       | **Tenat**                      | **Geo Fabric** |
| ---------------------------------- | ------------------------------ | -------------- |
| [Play](https://play.macrometa.io/) | `demo@macrometa.com` | `e-commerce-search-flow` |

## Deployment

| **Source Code**                       | **Deployment**                      | 
| ---------------------------------- | ------------------------------ | 

> | [Github](https://github.com/Macrometacorp/e-commerce) | [Deployment](https://macrometacorp.github.io/e-commerce/) | 

Reviewer: ```suggestion
| [GitHub](https://github.com/Macrometacorp/e-commerce) | [Deployment](https://macrometacorp.github.io/e-commerce/) | 
```

Reviewer: Private repo.

## Solution

> Experience the full potential of Macrometa's advanced full-stack capabilities with our innovative e-commerce demo. Discover a fully functional e-commerce website that offers personalized experiences to every visitor. Our cutting-edge technology tracks and analyzes visitor actions, enabling us to provide tailored recommendations based on their preferences. Furthermore, our e-commerce store boasts real-time stock updates and features a sophisticated search engine, ensuring a seamless and efficient shopping experience.

Reviewer: ```suggestion
Experience the full potential of Macrometa's advanced full-stack capabilities with our innovative e-commerce demo. This sample application is a fully functional e-commerce website that offers personalized experiences to every visitor. Using the Macrometa platform, it tracks and analyzes visitor actions to provide tailored recommendations based on their preferences. The e-commerce store has real-time stock updates and features a sophisticated search engine, ensuring a seamless and efficient shopping experience.
```

## Components

This demo includes the following components:

> - Personalized recommendations - Macrometa Stream Workers read visitor actions from a Collection Stream, creating a list of recommended products for each visitor.

Reviewer: Nothing in this list should be capitalized except Macrometa and the first word in the sentence or definition term. Remove GDN.

- Real-time stock updates - Macrometa Stream Workers read product updates from a Collection Stream, updating the product stock in real-time.

> - Search - Macrometa GDN Search service enables full-text search for our storefront, enabling users to find products based on a variety of terms including product name, brand and category.

Reviewer: ```suggestion
- Search - Macrometa GDN Search service enables full-text search for our storefront, enabling users to find products based on a variety of terms including product name, brand, and category.
```

- Serverless backend – Macrometa Query Workers provides an interface between the web app and the backend, providing ultra-low latency data and compute.

## Visitor Personalization

> Macrometa GDN Streams and Query Workers enable real-time visitor personalization, including recommendations and real-time stock updates.

Reviewer: ```suggestion
Macrometa streams and query workers enable real-time visitor personalization, including recommendations and real-time stock updates.
```

> - **Tailored Product Recommendations**

Reviewer: Remove the bold.
